talks/Friday/PULPO - optimization and brightway/environment.yml

```diff
@@ -0,0 +1,14 @@
+name: pulpo_brightcon_env
+channels:
+  - conda-forge
+dependencies:
+  - python=3.10
+  - pip
+  - ipopt=3.11.1
+  - pip:
+      - pulpo-dev
+      - python-louvain
```

Comment:
The latest versions of networkx ipywidgets ipympl pdf2image are all on conda-forge. Why get them via pip?